import/xsl/dspace-xoai.xsl

```diff
@@ -0,0 +1,164 @@
+   <xsl:output method="xml" indent="yes" encoding="utf-8"/>
+   <xsl:param name="collection">DSpace</xsl:param>
+   <xsl:param name="urlPrefix">http</xsl:param>
+   <xsl:param name="geographic">false</xsl:param>
+   <xsl:param name="id_tag_name">identifier</xsl:param>
+   <xsl:param name="change_tracking_core">biblio</xsl:param>
+   <xsl:param name="change_tracking_date_tag_name"></xsl:param>
+   <xsl:param name="workKey_include_regEx"/>
+   <xsl:param name="workKey_exclude_regEx"/>
+   <xsl:param name="workKey_transliterator_rules">:: NFD; :: lower; :: Latin; :: [^[:letter:] [:number:]] Remove; :: NFKC;</xsl:param>
+   <xsl:template match="/">
+      <add>
+         <doc>
+	    <!-- RECORD ID -->
+	    <field name="id">
+               <xsl:value-of select="//*[name()=$id_tag_name]"/>
+            </field>
+            <!-- RECORD FORMAT -->
+	    <field name="record_format">dspace</field>
+            <!-- INSTITUTION -->
+	    <field name="institution">
+		<xsl:if test="//*[@name='publisher']">
+	            <xsl:value-of select="//*[@name='publisher']/*/*[@name='value']"/>
+	        </xsl:if>
+            </field>
+            <!-- COLLECTION -->
+            <field name="collection">
+               <xsl:value-of select="$collection"/>
+            </field>
+            <!-- TITLE -->
+            <xsl:if test="//*[@name='title']">
+               <field name="title">
+	          <xsl:value-of select="//*[@name='title']/*/*[@name='value']"/>
+               </field>
+               <field name="title_short">
+		  <xsl:value-of select="//*[@name='title']/*/*[@name='value'][normalize-space()]"/>
+               </field>
+               <field name="title_full">
+		  <xsl:value-of select="//*[@name='title']/*/*[@name='value'][normalize-space()]"/>
+               </field>
+               <field name="title_sort">
+		  <xsl:value-of select="php:function('VuFind::stripArticles', string(//*[@name='title']/*/*[@name='value'][normalize-space()]))"/>
+               </field>
+            </xsl:if>
+            <!-- AUTHOR -->
+	    <xsl:for-each select="//*[@name='dc']/*[@name='contributor']/*[@name='author']/*/*[@name='value']">
+               <xsl:if test="normalize-space()">
+                  <field name="author">
+		      <xsl:value-of select="normalize-space()"/>
+                  </field>
+                  <!-- use first author value for sorting -->
+                  <xsl:if test="position()=1">
+                     <field name="author_sort">
+                        <xsl:value-of select="normalize-space()"/>
+                     </field>
+                  </xsl:if>
+               </xsl:if>
+            </xsl:for-each>
+            <!-- CO AUTHOR -->
+	    <xsl:for-each select="//*[@name='contributor']/*[@name='advisor']/*[@name='none']/*[@name='value']">
```

Comment:
Is "advisor" the only possible value here that could apply to author2?

Reply:
Hi @demiankatz I'm trying to find the answer to this. On the AUT DSpace instance it does look like 'advisor' is the only possible value here. I am checking for a wider DSpace context if that is the case.